server/msteams/client.go

```diff
@@ -547,7 +575,11 @@ func (tc *ClientImpl) UploadFile(teamID, channelID, filename string, filesize in
 		var chatFolder models.DriveItemable
 		var cErr error
 		folderName := "Microsoft Teams Chat Files"
-		itemID = *rootDirectory.GetId() + ":/" + folderName
+		rootDirectoryID := ""
+		if rootDirectory.GetId() != nil {
+			rootDirectoryID = *rootDirectory.GetId()
```

Comment:
Here's an example of a case where I don't know what happens if we keep going below with a `rootDirectoryID == ""`. Should we just return an error in this case? Or do we have to be smarter below in realizing there is no root directory and so don't try to create the chat folder?